src/commands.cpp

```diff
@@ -5450,6 +5450,25 @@ void CommandConfirmCancelLink::procresult()
     }
 }
 
+CommandResendVerificationEmail::CommandResendVerificationEmail(MegaClient *client)
+{
+    cmd("era");
+    tag = client->reqtag;
+}
+
+void CommandResendVerificationEmail::procresult()
+{
+    if (client->json.isnumeric())
+    {
+        return client->app->resendverificationemail_result((error)client->json.getint());
```

Comment:
This method returns `void`. Actually, you can leave it fall through, and it will return anyway (as it is the code today).